core/templates/dev/head/components/state-editor/state-hints-editor/state-hints-editor.directive.ts

```diff
@@ -286,6 +286,8 @@ angular.module('oppia').directive('stateHintsEditor', [
             StateHintsService.saveDisplayedValue();
             $scope.onSaveHints(StateHintsService.displayed);
           };
+
+          StateEditorService.updateStateHintsEditorInitialised(true);
```

Comment:
Why do we need this over here? (I'm expecting stateHintsEditorInitialised to be true before and after clicking "Save Hints" button, am I missing something?)

Reply:
stateHintsEditorInitialised indicates that the state hints editor has been initialised (ie. the component has loaded and registered any event listeners it may have, in this case it registers 'stateEditorInitialized').